api/views.py

```diff
@@ -147,4 +147,35 @@ def get_user_events(request, user_id):
                 ]
             }
         })
-    return JsonResponse({'error': 'Invalid request method'}, status=405)
\ No newline at end of file
+    return JsonResponse({'error': 'Invalid request method'}, status=405)
+
+@csrf_exempt
+def get_one_event(request, user_id, event_id):
+    if request.method == 'GET':
+        user = get_object_or_404(User, id=user_id)
+
+        event = get_object_or_404(Event, id=event_id)
+
+        e_1 = Event.objects.filter(owner=user, id=event.id)
```

Comment:
You figured out the filter thats awesome

Reply:
Nice!